### PR TITLE
Check compatibility with berkeley

### PR DESCRIPTION
**Pull request by psteckler**
For PRs against `develop`, check that we can sync a daemon against a `berkeley` seed daemon.

### CODE REVIEW

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by psteckler** [comment]
> Can this one also replace the existing `connect to berkeley` test?

The test here connects to a local daemon built with the `berkeley` branch, not the `berkeley` testnet. So they're different tests.

File: buildkite/scripts/check-compatibility.sh [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+echo "${MAINLINE_BRANCH} seed done bootstrapping"
+
+# generate PR libp2p keypair
+gen_libp2p_keypair $PR_IMAGE_ID "pr_docker"
+
+PR_COMMITTED_IMAGE_ID=$COMMITTED_IMAGE_ID
+
+echo "Booting PR daemon"
+
+boot_and_sync $PR_COMMITTED_IMAGE_ID 8305 3086 $MAIN_BRANCH_LIBP2P_PEER_ID 8302
```
[the remark this answers is not in the document]
Comment (psteckler) [reply buildkite/scripts/check-compatibility.sh:203]: The value of `COMMITTED_IMAGE_ID` changes after each call to `gen_libp2p_keypair`, so `MAIN_BRANCH_COMMITTED_IMAGE_ID` is distinct from `PR_COMMITTED_IMAGE_ID`.

**Pull request merged**